### DISCUSSION TITLE
How can we execute Relay-style cursor paginated queries?

### DISCUSSION THREAD

**Discussion by daniellin215** [TanStack/query#3130] [Q&A] [answered]
Just looked at the docs and it seems like only the page and offset pagination is mentioned, but I was wondering if anyone with GraphQL uses cursor based pagination and react-query? Thanks!

**Comment by TkDodo** [comment, marked as answer]
the infinite query example uses cursor-based pagination: https://react-query.tanstack.com/examples/load-more-infinite-scroll

**Reply by daniellin215** [reply]
hey @TkDodo , thanks for the reply. For others that may stumble upon this I also found some examples here (incl. graphql): https://stackoverflow.com/questions/62129771/react-query-how-to-paginate-with-useinfinitequery
